ecl/network/v2/static_routes/testing/request_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	th "github.com/nttcom/eclcloud/testhelper"
 )
 
-func TestList(t *testing.T) {
+func TestListStaticRoute(t *testing.T) {
```

Comment:
```suggestion
func TestListStaticRoutes(t *testing.T) {
```

Reply:
Thank you. this suggestion is supported.